furusiya/model/entities/party/player.py

```diff
@@ -33,9 +33,19 @@ def __init__(self):
         self.level = 1
         self.stats_points = 0
         self.arrows = config.data.player.startingArrows
+        self.mounted = False
 
         print("You hold your wicked-looking {} at the ready!".format(weapon_name))
 
+    def mount(self, horse):
```

Comment:
Reminds me of ruby madness I coded once: please gate these two methods behind an `if self.features.horseIsMountable` feature check.